src/node_api/model/cli.rs

```diff
@@ -33,9 +33,15 @@ pub struct RequestDockerBuild {
     pub image: String,
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+pub struct TransparentLogOutputParams {
+    pub format: Option<String>,
+}
+
 #[derive(Debug, Deserialize, Serialize)]
 pub struct RequestDockerLog {
     pub image: String,
+    pub output_params: Option<TransparentLogOutputParams>,
```

Comment:
I would not use `Option` here, but directly use the `TransparencyLogOutputParams`:
```suggestion
    pub output_params: TransparentLogOutputParams,
```

Keep the logic of handling the optional provisioning of the `format` argument in the CLI code.